backend/server/src/serve_frontend.rs

```diff
@@ -0,0 +1,68 @@
+use actix_web::{
+    get,
+    http::{header::ContentType, StatusCode},
+    web::ServiceConfig,
+    HttpRequest, HttpResponse, Responder,
+};
+use mime_guess::{from_path, mime};
+use rust_embed::RustEmbed;
+
+#[derive(RustEmbed)]
+// Relative to server/Cargo.toml
+#[folder = "../../frontend/packages/host/dist/"]
```

Comment:
```suggestion
#[folder = "../../frontend/host/dist/"]
```

`packages` dir was removed as part of the typescriptification... if we're going to our rust backend perhaps one could argue for the return of it though!

Reply:
3f9cd65